admin/auth_token.go

```diff
@@ -26,19 +29,30 @@ func (t *userAuthToken) Token() *authtoken.Token {
 }
 
 func (t *userAuthToken) OwnerID() string {
+	if t.model.UserID != t.model.RepresentingUserID {
+		return t.model.RepresentingUserID
+	}
+
 	return t.model.UserID
 }
 
 // IssueUserAuthToken generates and persists a new auth token for a user.
-func (s *Service) IssueUserAuthToken(ctx context.Context, userID, clientID, displayName string) (AuthToken, error) {
+func (s *Service) IssueUserAuthToken(ctx context.Context, userID, clientID, displayName, representingUserID string, expirationTTLMinutes int64) (AuthToken, error) {
```

Comment:
1. `ttl *time.Duration` instead of `expirationTTLMinutes int64`
2. `representingUserID *string` (pointer)